core/src/Cache/ITokenCacheAccessor.cs

```diff
@@ -31,29 +31,29 @@ namespace Microsoft.Identity.Core.Cache
 {
     internal interface ITokenCacheAccessor
```

Comment:
Please add comments. If you add comments on the interface methods, you do not have to add them on the implemeting class.

Reply:
pls. take in the code style discussion

---
In reply to: [199545504](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1108#discussion_r199545504) [](ancestors = 199545504)